python/tk_multi_publish2/publish_tree_widget/publish_tree_widget.py

```diff
@@ -65,6 +65,12 @@ def __init__(self, parent):
 
         # forward double clicks on items to the items themselves
         self.itemDoubleClicked.connect(lambda i, c: i.double_clicked(c))
+        
+        # workaround to make the scrollbar work properly for QT versions < 5 on macOS
```

Comment:
the comment explains what the code is doing, which is helpful and great! Could you also add one or two more lines explaining why we need to have this fix here? That would help our developers and users looking at the code! Your pull request comment is really informative, i suggest tweaking that and adding it into the code.